inc/hash_table.h

```diff
@@ -1,29 +1,24 @@
+#include "my_malloc.h"
+
 #ifndef HASH_TABLE_H  
 #define HASH_TABLE_H
 
 #include <stdio.h>
 #include <string.h>
 #include <stdint.h>
 #include <stdbool.h>
-#include <stdlib.h>
-#include "my_malloc.h"
 
-typedef uint64_t (hash_function)(const char *input, size_t);
-typedef struct hash_table_s t_hash;
+#include <stdlib.h>
+#include <math.h>
 
-struct hash_table_s {
-    uint32_t size;
-    hash_function *hash;
-    t_block **elements;
-};
```

Comment:
was running into some circular dependency issues so moved this into my_malloc.h which clear things up